src/Controllers/EditPageController.php

```diff
@@ -0,0 +1,37 @@
+<?php
+namespace App\Controllers;
+use App\Models\TasksModel;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Psr\Http\Message\ResponseInterface as Response;
+/**
+ * Class EditPageController
+ * @package App\Controllers
+ * @return Response with view
+ */
+class EditPageController {
+    protected TasksModel $model;
+    protected $view;
+    /**
+     * EditPageController constructor.
+     * @param TasksModel $model
+     * @param $view
+     */
+    public function __construct(TasksModel $model, $view)
```

Comment:
The view can be typehinted as `PhpRenderer` with a `use Slim\Views\PhpRenderer;` declaration